observability/logging/logging-6.0/log6x-about.adoc

```diff
@@ -5,3 +5,46 @@ include::_attributes/common-attributes.adoc[]
 :context: logging-6x
 
 toc::[]
+
+The `ClusterLogForwarder` custom resource is the central configuration point for OpenShift Logging 6.0. It defines the desired state of your logging system, including inputs, outputs, and pipelines. The Cluster Logging Operator watches for changes to this resource and automatically applies the necessary configurations to the underlying logging components.
+
+== Service account
+This release of Cluster Logging requires administrators to explicitly grant log collection permissions to the service account associated with *ClusterLogForwarder*. This was not required in previous releases for the legacy logging scenario consisting of a *ClusterLogging* and, optionally, a *ClusterLogForwarder.logging.openshift.io* resource.
+
+To use the existing service account (that is, `logcollector`) from a previous release, create the following *ClusterRoleBinding*:
+
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user collect-application-logs -z openshift-logging:logcollector
+$ oc adm policy add-cluster-role-to-user collect-infrastructure-logs -z openshift-logging:logcollector
+----
+
+Additionally, create the following *ClusterRoleBinding* if collecting audit logs:
+
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user collect-audit-logs -z openshift-logging:logcollector
+----
+
+== Inputs and Outputs
+
+Inputs specify the sources of logs to be forwarded. OpenShift Logging 6.0 provides built-in input types such as `application`, `infrastructure`, and `audit`, which select logs from different parts of your OpenShift cluster. You can also define custom inputs based on namespaces or pod labels to fine-tune log selection.
+
+Outputs define the destinations where logs are sent. Various output types are supported, including Elasticsearch, Fluentd, Syslog, Kafka, Loki, and CloudWatch. Each output type has its own set of configuration options, allowing you to customize the behavior and authentication settings.
```

Comment:
including azureMonitor, cloudwatch, elasticsearch,http,kafka,loki,lokiStack,googleCloudLogging,splunk,syslog and otlp.